torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -200,6 +206,7 @@ bool validateKernelArgScalar(
     case c10::ScalarType::Bool:
       match = param_type == DataType::Bool;
       break;
+    // TODO: support complex double scalar
```

Comment:
I guess we'd need another scalar IR node for this, could consider doing this in another PR.

Reply:
Yeah, I will extend complex support for scalar in a separate PR.